harper-core/affixes.json

```
@@ -633,6 +633,16 @@
       "gifts_metadata": {
         "article": true
       }
+    },
+    "j": {
+      "#": "adverb property",
+      "suffix": true,
+      "cross_product": true,
+      "replacements": [],
+      "adds_metadata": {},
+      "gifts_metadata": {
+        "adverb": {}
```

All others are like this


```suggestion
        "adverb": true
```

Previously reported here

https://github.com/Automattic/harper/pull/608#discussion_r1949814237

---

@elijah-potter you merged apparently

---

Ah, no. The `{}` is intentional.

---

OK thanks